src/BenchmarkDotNet/Running/BenchmarkRunnerClean.cs

```diff
@@ -84,6 +85,26 @@ internal static Summary[] Run(BenchmarkRunInfo[] benchmarkRunInfos)
 
                     foreach (var benchmarkRunInfo in supportedBenchmarks) // we run them in the old order now using the new build artifacts
                     {
+                        if (benchmarkRunInfo.Config.Options.IsSet(ConfigOptions.Resume))
+                        {
+                            var directoryInfo = new DirectoryInfo(rootArtifactsFolderPath);
+                            var lastFiles = directoryInfo.GetFiles($"{title.Split('-')[0]}*");
+                            if (lastFiles.Count() > 1)
+                            {
+                                var lastUpdatedFile = lastFiles[lastFiles.Length - 2];
```

Comment:
You need to sort by date for this to work. `.OrderBy(p => p.CreationTime)`

Reply:
@timcassell thanks. I already committed your suggestion.